src/services/ui/src/routes.ts

```diff
@@ -0,0 +1,10 @@
+/** TODO: Implement enum values where `to` or `href` is currently just a string. */
+export enum ROUTES {
```

Comment:
To prevent circular imports between `router.tsx` and the components/pages using `ROUTES`, they've been broken out into their own file -- this can be something of an "all constants" file in the future if we need it. Without this separation, the circular dependency between files leads to an `undefined` error when importing and using the constant values.